lib/vagrant/machine.rb

```diff
@@ -449,7 +449,10 @@ def ssh_info
       info[:host] = @config.ssh.host if @config.ssh.host
       info[:port] = @config.ssh.port if @config.ssh.port
       info[:keys_only] = @config.ssh.keys_only
-      info[:paranoid] = @config.ssh.paranoid
+      info[:verify_host_key] = @config.ssh.verify_host_key || @config.ssh.paranoid
+      if @config.ssh.paranoid
+        @env.ui.warn(I18n.t("vagrant.config.ssh.paranoid_deprecated"))
+      end
```

Comment:
Not really sure if this is the right place for the warning..but it might be? It's fairly noisy here however if a user still has the `:paranoid` setting in their config.

Reply:
The warning would be better to originate from the config. We can also set the `verify_host_key` value within the config from `paranoid` if it is set within the config as well (when finalizing) and fully remove the `paranoid` usage from this file.